src/Razor/src/Microsoft.VisualStudio.LanguageServices.Razor/Microsoft.VisualStudio.LanguageServices.Razor.csproj

```diff
@@ -158,6 +158,9 @@
       <DependentUpon>ProjectSystem\Rules\RazorComponentWithTargetPath.xaml</DependentUpon>
     </Compile>
   </ItemGroup>
+  <ItemGroup>
+    <Folder Include="LanguageClient\ProjectSystem\" />
+  </ItemGroup>
```

Comment:
Remove?

Reply:
Yep, added a folder via VS :) Didn't notice this, thanks.